### DISCUSSION TITLE
After toggling fullscreen, video stops and it shows background color

### DISCUSSION THREAD

**Discussion by kangchals** [vimeo/player.js#999] [Bug reports]
### Expected Behavior

Player should be played normally on toggling fullscreen.

### Actual Behavior

- player is stopped and it shows white screen ( it seems that it is default background color )
- In addition error message should not be shown in the console
  - `locked a frame with origin "xxx" from accessing a frame with origin "https://player.vimeo.com". Protocols, domains, and ports must match.`
- it happens only on desktop safari(15.3, 15.5) browser.
- in addition, icon is wrong, below icon is showing when it is fullscreen.
![image](https://user-images.githubusercontent.com/7294289/174270158-d41c8933-07eb-413d-bfa8-5e09a93ac832.png)
 

### Steps to Reproduce

1, enter to a page which has the video
2, when the page is loaded, video will be fullscreen
3, click the play button
4, video plays
5, click fullscreen button
6, video will be shown with normal size and video is stopped and it is showing white screen.

**Comment by bdougherty** [comment]
@kangchals are you still seeing this with the latest Safari?
